Add Note for using jsonarray in GenerateJson (#98072)
Azure/azure-content: articles/active-directory-b2c/json-transformations.md

       }
    }
    ```

> > [!NOTE]
> > The GenerateJson claims transformation accepts plain strings. If an input claim contains a JSON string, that string will be escaped. In the following example, if you use email output from CreateJsonArray above that is ["someone@contoso.com"] and used it as input parameter in the GenerateJson example, the email will look like the following JSON:
> - Output claim:
>   - **requestBody**:
> 
>     ```json
>     {
>        "customerEntity":{
>           "email":"[\"someone@contoso.com\"]",
>           "userObjectId":"01234567-89ab-cdef-0123-456789abcdef",
>           "firstName":"John",
>           "lastName":"Smith",
>           "role":{
>              "name":"Administrator",
>              "id": 1
>           }
>        }
>     }
>     ```

Reviewer: ```suggestion

The **GenerateJson** claims transformation accepts plain strings. If an input claim contains a JSON string, that string will be escaped. In the following example, if you use email output from [CreateJsonArray above](json-transformations.md#example-of-createjsonarray), that is ["someone@contoso.com"], as an input parameter, the email will look like as shown in the following JSON claim:

- Output claim:
  - **requestBody**:

    ```json
    {
       "customerEntity":{
          "email":"[\"someone@contoso.com\"]",
          "userObjectId":"01234567-89ab-cdef-0123-456789abcdef",
          "firstName":"John",
          "lastName":"Smith",
          "role":{
             "name":"Administrator",
             "id": 1
          }
       }
    }
    ```
```

Reviewer: @bernawy,

Please see my proposed commit. Thank you.
